content/en/docs/concepts/extend-kubernetes/api-extension/custom-resources.md

```diff
@@ -162,7 +162,7 @@ API resource allows you to define custom resources.
 Defining a CRD object creates a new custom resource with a name and schema that you specify.
 The Kubernetes API serves and handles the storage of your custom resource.
 The name of a CRD object must be a valid
-[DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
+[DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names) (see [how to create a CRD](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions#create-a-customresourcedefinition) for more details). The name of an object whose kind/resource is defined by a CRD is also constrained to be a valid DNS subdomain name.
```

Comment:
Instead of adding the sentence here, can we add a row to the table in content/en/docs/concepts/extend-kubernetes/api-extension/custom-resources.md for "Instance Name" where CRD is restricted to DNS subdomain and aggregated API can define their own rules.

The content is technically correct, but I think it fits the comparison table very well.

Reply:
@deads2k: Do you mean the table in the "Advanced features and flexibility" section?

Reply:
I have force-pushed a revision along the lines that @deads2k suggested.